scripts/gen_release_requirements.py

```
@@ -0,0 +1,160 @@
+    "platform_release": "10",
+    "platform_system": "Windows",
+    "platform_version": "10.0.19045",
+    "python_full_version": "3.12.10",
+    "python_version": "3.12",
+    "implementation_name": "cpython",
+}
+
+HEADER = """\
+# This file was autogenerated from uv.lock via the following command:
+#    python scripts/gen_release_requirements.py
+# NOTE: 发布裁剪 - pyside6 / pyside6-addons（含 QtWebEngine，约 200MB）已从本文件
```

Ruff (0.16.1)

[warning] String contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF001)